### PR TITLE
Fix(toc): missing caret icon in web components

### PR DESCRIPTION
**Pull request by mmiask**
Resolves: #987

### CODE REVIEW

File: packages/elements-web-components/src/index.ts [hunk 1]
```diff
@@ -1,4 +1,10 @@
+import { dom, library } from '@fortawesome/fontawesome-svg-core';
+import { faChevronDown, faChevronRight, faCube } from '@fortawesome/free-solid-svg-icons';
```
Comment (marcelltoth) [inline packages/elements-web-components/src/index.ts:2]: Do we list these as `devDependencies` of `elements-web-components`?

**Review by marcelltoth — APPROVED** [review]
Beyond that 1 question, looks great now! Good work @mmiask !

File: packages/elements-web-components/src/index.ts (hunk 1 above)
Comment (mmiask) [reply packages/elements-web-components/src/index.ts:2]: No, these are direct dependencies in `elements`

Comment (mmiask) [reply packages/elements-web-components/src/index.ts:2]: @marcelltoth Should we?

**Pull request merged**